azure-pipelines.yml

```diff
@@ -23,9 +23,13 @@ variables:
     value: "1.0."
   - name: UKHOAssemblyProduct
     value: "File Share Desktop Client"
+  - name: WindowPool
+    value: "NautilusBuild"
   - name: coverityPool
     value: NautilusBuild
   - group: Covscan-vars
+  - name: DotNetSdkVersion #for tasks without global.json context
```

Comment:
Not used any where in the pipeline.

Reply:
I think it needs to be passed in UseDotNet@2 task as version input parameter

Reply:
Updated as sugested